agent/containers/images/visualizers/run.py

```diff
@@ -76,12 +76,22 @@
 headers["Authorization"] = f"Bearer {token}"
 
 metric_type = os.environ["VIS_TYPE"]
+
```

Comment:
This comment is meant for line 33 above:  I think we only want to start our own Redis server if an external one is not provided.